README.md

```diff
@@ -16,50 +18,66 @@
   * [CMake](#optional-cmake)  
   * [DOxygen](#optional-doxygen)  
 
-<a name="introduction"/>
-
 ## Introduction
-The MAME development environment for Windows consists of the GCC compiler (by way of MinGW), and the MSYS2 (POSIX/Unix compatability layer), plus various utilities such as Python and Git. It is available as a prepackaged installer, or can be assembled from upstream components. 
+<a name="introduction"/>
+The MAME development environment for Windows consists of the GCC compiler (by way of MinGW), and the MSYS2 POSIX/Unix compatibility layer, plus various utilities such as Python and Git. It is available as a prepackaged installer, or can be assembled from upstream components. 
 
-Source control is handled by github (***https://github.com/mamedev/mame.git***), so you'll need to check out a copy.
+Source control is handled by GitHub (***https://github.com/mamedev/mame.git***), so you'll need to check out a copy.
 Various modules are disabled by default, but can be enabled through arguments when building and may require additional MSYS2 packages to be installed.
 
-<a name="installandbuild"/>
-
 ## Installation and building
+<a name="installandbuild"/>
 
 <a name="downloads"/>
 
 ### Downloads
-* Dual 32-bit/64-bit - [msys64-32-2019-12-23.exe](https://github.com/mamedev/buildtools/releases/download/5.0/msys64-32-2019-12-23.exe) _(If you have 64-bit Windows but wish to build for both 32-bit and 64-bit. To switch between different mingw versions use **config32.bat** and **config64.bat**)_
+* [Latest 64-Bit Tools Download](https://github.com/mamedev/buildtools/releases/latest)
 
+### Installation
 <a name="installation"/>
 
-### Installation
-If you are installing it in a location other than the default (*c:\msys64* or *c:\msys32*), after you unpack double-click : **autorebase.bat**
+If you are installing it in a location other than the default (*C:\msys64* or *C:\msys32*), after you unpack double-click ``autorebase.bat``
```

Comment:
That’s already outdated – the staged location for building the tools is `C:\Users\Public\msys64` and we don’t have a 32-bit MSYS2 any more.

Reply:
I was expecting that to be brought up, and decided to wait and see what you'd say before making much in the way of changes. I'll get that in tonight.

Reply:
Okay, let me know if there are further issues and I'll look at things as needed.